jekyll/_cci2/collect-test-data.md

```diff
@@ -470,17 +470,9 @@ A working `.circleci/config.yml` section might look like this:
 #### Jest
 {:.no_toc}
 
-To collect Jest data, first create a Jest config file called `jest.config.js` with the following:
```

Comment:
https://github.com/circleci/circleci-docs.wiki.git

Reply:
@chanmya13C that link doesn't work for me, was there something I should do with this? thanks!

Reply:
Sorry, I believe the above comment is spam.